ddl/column.go

```diff
@@ -1423,13 +1423,13 @@ func checkAndApplyAutoRandomBits(d *ddlCtx, m *meta.Meta, dbInfo *model.DBInfo,
 
 // checkNewAutoRandomBits checks whether the new auto_random bits number can cause overflow.
 func checkNewAutoRandomBits(idAccessors meta.AutoIDAccessors, oldCol *model.ColumnInfo,
-	newCol *model.ColumnInfo, newAutoRandBits uint64, tblInfoVer uint16) error {
-	newLayout := autoid.NewShardIDLayout(&newCol.FieldType, newAutoRandBits)
+	newCol *model.ColumnInfo, newAutoRandBits uint64, tblInfo *model.TableInfo) error {
```

Comment:
A little bit weird for the param, but acceptable.

Reply:
The parameters are updated.